GAME/HAIR.C

```diff
@@ -40,7 +40,7 @@ void InitialiseHair()//3AC70, 3B170
 			hptr->pos.y_pos = bone[2];
 			hptr->pos.z_pos = bone[3];
 
-			hptr->pos.x_rot = -16384;
+			hptr->pos.x_rot = -ANGLE90;
```

Comment:
I'm not too fond of moving angular/positional constants to definitions. It may not be the best idea for future potential modifications.